Src/ST-LIB_LOW/Sensors/LinearSensor/LinearSensor.cpp

```diff
@@ -0,0 +1,26 @@
+#include "ST-LIB_LOW/Sensors/LinearSensor/LinearSensor.hpp"
+#include "ADC/ADC.hpp"
+
+LinearSensor::LinearSensor(Pin pin, double slope, double offset, double *value)
+: slope(slope), offset(offset), value(value){
+	optional<uint8_t> identification = ADC::register_adc(pin);
+	if(identification){
+		id = identification.value();
+		ADC::turn_on_adc(id);
+	}else{
```

Comment:
```suggestion
	}
	else {
```
Se consistente con el formato

Reply:
Como esta esta bien